### PR TITLE
add missing dependencies and compile check test

### PR DESCRIPTION
**Pull request by aaannz**

### CODE REVIEW

**Comment by aaannz** [comment]
hmm.. still not running compile checks, hold on

File: t/Makefile.am [hunk 1]
```diff
@@ -1,3 +1,3 @@
-TESTS = 01-test_needle.pl 02-test_ocr.pl 03-testapi.pl
+TESTS = 00-compile-check-all.t 01-test_needle.pl 02-test_ocr.pl 03-testapi.pl
```
Comment (coolo) [inline t/Makefile.am]: it's .pl in here

Comment (aaannz) [inline t/Makefile.am]: Yeah, I renamed it just now and forgot about that.

**Comment by aaannz** [comment]
Ok, this is taking a lot more time I expected.
The problem is that tinycv is compiled using system perl libraries instead of perlbrew ones. The -I from configure correctly points to perlbrew libs, but perl binary is used system one. That may be one culprit. Or MakeMaker creates Makefile using system sources, dunno.

@coolo any ideas?

**Comment by coolo** [comment]
I fixed and merged it

**Pull request closed without merging**